CONTRIBUTING.md

```diff
@@ -59,8 +59,11 @@ one of the following forms:
    ([example](https://github.com/void-linux/void-packages/commit/176d9655429188aac10cd229827f99b72982ab10)).
 - for package updates, use `<pkgname>: update to <version>.`
    ([example](https://github.com/void-linux/void-packages/commit/b6b82dcbd4aeea5fc37a32e4b6a8dd8bd980d5a3)).
-- for template modifications without a version change, use `<pkgname>: <reason>`
+- for template modifications without a version change, including removals, use
+   `<pkgname>: <reason>`
```

Comment:
removes should be rather `<pkgname>: remove package`, reason possibly in body.

Reply:
I will add an example for it, then!